eng/pipelines/azure-pipelines-internal.yml

```diff
@@ -97,22 +98,20 @@ extends:
         SDLValidationParameters:
           enable: false
 
-    - template: /eng/pipelines/arcade/stage-unit-tests.yml@self
-      parameters:
-        jobMatrix:
-        - name: win_unit_tests
-          displayName: Windows Unit Tests
-          pool:
-            name: Azure Pipelines
-            vmImage: $(HostedWindowsImage)
-            os: windows
-          timeout: 120
-          testOS: Windows
-        - name: mac_unit_tests
-          displayName: macOS Unit Tests
-          pool:
-            name: Azure Pipelines
-            vmImage: $(HostedMacImage)
-            os: macOS
-          timeout: 120
-          testOS: macOS
+    - ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
We need to find a way to not update the build number when running these stages and also publish assets